commands/pull_request.go

```diff
@@ -230,14 +236,23 @@ func pullRequest(cmd *Command, args *Args) {
 Write a message for this pull request. The first block
 of text is the title and the rest is the description.`, fullBase, fullHead))
 
-	if cmd.FlagPassed("message") {
+	switch {
```

Comment:
Any strong reason this `if` chain got converted to a `switch`? I'd like to keep the diff as minimal as possible